cmd/telemetrygen/internal/metrics/config.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/telemetrygen/internal/common"
+)
+
+// Config describes the test scenario.
+type Config struct {
+	common.Config
+	NumMetrics int
+}
+
+// Flags registers config flags.
+func (c *Config) Flags(fs *pflag.FlagSet) {
+	fs.IntVar(&c.WorkerCount, "workers", 1, "Number of workers (goroutines) to run")
```

Comment:
Could you try to move the flags of common config into package common? Or record an issue to track this?

Reply:
Sure, done :)